src/Template/Layout/default.ctp

```diff
@@ -2,7 +2,7 @@
 <html lang="<?= \Locale::getPrimaryLanguage(\Cake\I18n\I18n::locale()) ?>">
 <head>
     <?= $this->Html->charset(); ?>
-    <title><?= $this->get('title');?></title>
+    <title><?= $this->get('siteTitle');?></title>
```

Comment:
It currently uses the old default if there isn't a `siteTitle` in order to allow users to migrate. I'm not going to revert.